inspirehep/modules/warnings/ext.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Extension that suppresses warnings.
+
+By default Invenio-Base enables deprecation warnings. This extension
+disables them, unless ``DEBUG_SHOW_DEPRECATION_WARNINGS`` is ``True``.
+
+This extension also disables SQLAlchemy warnings, unless the variable
+``DEBUG_SHOW_SQLALCHEMY_WARNINGS` is set to ``True``.
```

Comment:
@jacquerie are these warnings telling us something important or not? The fact that so far we have ignored them, doesn't mean that it was the right thing to do. I can imagine that some warnings are really just noise. But others might be the anticipation of some imminent disaster.
Until now we were not using stuff on production for real, so no harm was caused. But as we approach putting on Invenio 3 on production we need to become more proactive towards them. 
Can you list on Github which warnings we have at the moment that you think it's safe to ignore? Are there warnings we should instead act upon (either in inspire-next repo or on upstream Invenio packages)?